torch/csrc/jit/runtime/operator.cpp

```diff
@@ -53,6 +53,16 @@ struct OperatorRegistry {
     to_register.clear();
   }
 
+  const std::vector<std::shared_ptr<Operator>>& getOperatorsWithLockHeld(
+      Symbol name) {
+    registerPendingOperators();
+    static std::vector<std::shared_ptr<Operator>> empty;
+    auto it = operators.find(name);
+    if (it != operators.end())
+      return it->second;
+    return empty;
```

Comment:
Um any reason this? 
Empty std::vector creation shouldn't be expensive and probably shouldn't all point to 1 static variable for a whole host of reasons (like it getting by corrupted if anything is appended to it? It needs to be copied anyway so it's not even saving a whole lot.